democracylab/settings.py

```diff
@@ -100,8 +99,8 @@
 
 # TODO: Use     'django_seo_js.middleware.UserAgentMiddleware',
 MIDDLEWARE = [
-    'common.helpers.caching.DebugUserAgentMiddleware',
     'django.middleware.security.SecurityMiddleware',
+    'django_seo_js.middleware.UserAgentMiddleware',
```

Comment:
Should remove this and the dependency in https://github.com/DemocracyLab/CivicTechExchange/blob/4b420b49872edf9fc3a87828f829ccbba65eece0/requirements.txt#L32